src/modules/form/components/DynamicForm.tsx

```diff
@@ -91,25 +90,14 @@ export const DynamicFormWithoutHandlers = forwardRef<
       ref,
       () => ({
         SubmitForm: () => {
-          // onSubmit({ values: getCleanedValues(), confirmed: false, });
           onSubmit({ ...getValuesForSubmit(), confirmed: false });
         },
-        SaveIfDirty: () => {
-          if (!handlers.methods.formState.isDirty || props.locked) return;
+        SaveDraftForm: () => {
+          if (props.locked) return;
           handleSaveDraft();
```

Comment:
I don't _think_ we still need the locked check, but leaving it just in case there is something I'm not catching.